### PR TITLE
[13.0][mrp_multi_level] improve planned orders

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
13.0.1.4.1 (2020-03-26)
~~~~~~~~~~~~~~~~~~~~~~~
  * Add menu entry for planned orders
  * Add button to navigate from planned orders to linked manufacturing orders
  * Add action to convert planned orders to fixed
  * When changing the due date in a planned order the release date is recomputed

cc @LoisRForgeFlow @HviorForgeFlow

### CODE REVIEW

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

File: mrp_multi_level/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "MRP Multi Level",
-    "version": "13.0.1.3.2",
+    "version": "13.0.1.4.1",
```
Comment (HviorForgeFlow) [inline mrp_multi_level/__manifest__.py]: Why this version change?

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Include also changelog on HISTORY.rst

File: mrp_multi_level/views/mrp_planned_order_views.xml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                    <filter
+                        name='due_date_by_week'
+                        string="Due Date (By Week)"
+                        context="{'group_by':'due_date:week'}"
+                    />
+                    <filter
+                        name='due_date_by_month'
+                        string="Due Date (By Month)"
+                        context="{'group_by':'due_date:month'}"
+                    />
```
Comment (LoisRForgeFlow) [inline mrp_multi_level/views/mrp_planned_order_views.xml]: keep only one filter, and remove what it is in parenteris. In v13 you can select the date range by default in this type of filters.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Can you add a more meaninful commit message?

And once you are going to do changes, please remove manual version bumping

File: mrp_multi_level/__manifest__.py (hunk 1 above)
Comment (JordiBForgeFlow) [reply mrp_multi_level/__manifest__.py]: Because it adds new features

**Comment by JordiBForgeFlow** [comment]
@LoisRForgeFlow 

> Can you add a more meaninful commit message?
> 
> And once you are going to do changes, please remove manual version bumping

Fixed.

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Comment by LoisRForgeFlow** [comment]
@JordiBForgeFlow merge at your convenience (remember to use minor option)

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**